Source/WebCore/loader/cache/CachedResourceLoader.cpp

```diff
@@ -561,6 +562,11 @@ bool CachedResourceLoader::allowedByContentSecurityPolicy(CachedResource::Type t
     if (!contentSecurityPolicy)
         return true;
 
+    contentSecurityPolicy->setIsReportingToConsoleEnabled(canReportViolationAsConsoleMessage);
+    auto scope = makeScopeExit([&] {
+        contentSecurityPolicy->setIsReportingToConsoleEnabled(true);
```

Comment:
Shouldn't we restore the old value, whatever it was?  This assumes the old value was false.

Reply:
It is just a way to temporarily disable console reporting, so we should always set it back to true.